frontend/packages/console-shared/src/components/form-utils/__tests__/FormFooter.spec.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import { FormFooter } from '@console/shared';
+
+type FormFooterProps = React.ComponentProps<typeof FormFooter>;
+const componentFactory = (props: FormFooterProps) => <FormFooter {...props} />;
+describe('FormFooter', () => {
+  let wrapper: ShallowWrapper<any>;
+  let props: FormFooterProps;
+  const className = 'ocs-form-footer';
+  beforeEach(() => {
+    props = {
+      errorMessage: 'error',
+      submitLabel: 'Create',
+      resetLabel: 'Reset',
+      cancelLabel: 'Cancel',
+      handleReset: jest.fn(),
+      handleCancel: jest.fn(),
+      sticky: false,
+      disableSubmit: false,
+      isSubmitting: false,
+    };
+    wrapper = shallow(componentFactory(props));
+  });
+
+  it('should contain submit, reset and cancel button', () => {
+    const content = wrapper.children().at(0);
+    const submitButton = wrapper.find('[data-test-id="submit-button"]');
+    const resetButton = wrapper.find('[data-test-id="reset-button"]');
+    expect(content.children()).toHaveLength(3);
```

Comment:
Not sure this child walking is a good idea... highly brittle if we want to change the structure.

Recommend `.find(Button)` maybe? (if needed you can use `.filter('[data-test-id="btn"]')` on the `Button` find to check existence). 

I would also recommend you update (remove?) the first line of this test too:
`    const content = wrapper.children().at(0);`

Reply:
updated